### ISSUE TITLE
2b2t queue chat greyed out on Impact version 4.9.1 on 1.12.2 with Optifine

### ISSUE THREAD

**Issue by engrav** [ImpactDevelopment/ImpactIssues#2523]
Describe the bug
On the Impact version 4.9.1 Minecraft Version 1.12.2 with OptiFine_HD_U_F5 The 2b2t queue chat is completely greyed out as well as the tab hud

  To Reproduce
1. Join 2b2t.org
2. See error

  Screenshot
![Minecraft 1 12 2 2020-07-16 16-39-19_Moment](https://user-images.githubusercontent.com/68402355/87725568-21353600-c783-11ea-8bcb-b8cbe8b332b2.jpg)

- Impact Version: 4.9.1 on 1.12.2 with OptiFine_HD_U_F5
- Using Optifine? Yes
- Operating System: Windows 10 64bit

**Comment by dominikaaaa** [comment]
Please provide a list of all enabled modules and other mods you're using

**Comment by ScanmanTM** [comment]
Does this happen without optifine?

**Comment by engrav** [comment]
> Does this happen without optifine?

No, i installed the Impact 4.9.1 without optifine and it worked fine.

> Please provide a list of all enabled modules and other mods you're using

No impact modules affect the bug and im not using any other mods.
I tried playing around with the optifine settings and nothing fixed it.

**Comment by big-lip-bob** [comment]
I had a similar problem when i introduced Baritone in my 1.12.2 pack, might be a problem correlated to it rather than impact

**Comment by dominikaaaa** [comment]
I believe this is a Baritone (+ Optifine?) issue that affects impact(?)

I've had people report when using KAMI Blue that has bundled Baritone, it makes Impact render modules turn gray. Doesn't happen without Baritone included in the jar, it appears there's leaking GL states somewhere.

I haven't confirmed if Optifine affects it or not for my users.

**Comment by Dinglesplat** [comment]
I have no optifine and this is happening to me, it is really annoying.

**Comment by RotaryRevor** [comment]
Issue just happened to me. It only happens to me if i have optifine included. No matter the issue

**Comment by nestomagic** [comment]
Same happens to me even without optifine, any fix???
